data/org.freedesktop.impl.portal.PermissionStore.xml

```diff
@@ -178,6 +178,16 @@
       <arg name="ids" type="as" direction="out"/>
     </method>
 
```

Comment:
Can't comment on the line in question, but the interface description says

> This document describes version 2 of the permission store interface.

Which needs to change.